src/mono/sample/mbr/browser/Program_v1.cs

```diff
@@ -4,29 +4,32 @@
 using System;
 using System.Reflection;
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices.JavaScript;
 using MonoDelta;
 
 namespace Sample
 {
-    public class Test
+    public partial class Test
     {
-	static DeltaHelper replacer = DeltaHelper.Make ();
+        static DeltaHelper replacer = DeltaHelper.Make ();
 
         public static void Main(string[] args)
         {
             Console.WriteLine ("Hello, World!");
         }
 
+        [JSExport]
         [MethodImpl(MethodImplOptions.NoInlining)]
```

Comment:
I think you could remove all `MethodImplOptions.NoInlining` where you have `[JSExport]`